src/cmd_line.rs

```diff
@@ -159,14 +163,9 @@ impl Default for CmdLineSettings {
 pub fn handle_command_line_arguments(args: Vec<String>) -> Result<()> {
     let mut cmdline = CmdLineSettings::try_parse_from(args)?;
 
-    // The neovim_args in cmdline are unprocessed, actually add options to it
-    let maybe_tab_flag = (!cmdline.no_tabs).then(|| "-p".to_string());
-
-    cmdline.neovim_args = maybe_tab_flag
-        .into_iter()
-        .chain(mem::take(&mut cmdline.files_to_open))
-        .chain(cmdline.neovim_args)
-        .collect();
+    if cmdline._no_tabs {
+        cmdline.tabs = false;
+    }
```

Comment:
I had to check if clap would support this out of the box now, but apparently, it's still unsupported.
https://github.com/clap-rs/clap/issues/815. 

It looks like we might be able to refactor our code to something like that workaround though, and also deal with the options and environment variables at the same time. Maybe even autogenerate the config file support or something.

But that's not something for this PR.